### PR TITLE
tests(devtools): sync e2e

### PR DESCRIPTION
**Pull request by adamraine**

### CODE REVIEW

File: third-party/devtools-tests/e2e/lighthouse/navigation_test.ts [hunk 1]
```diff
@@ -31,6 +38,9 @@ describe('Navigation', async function() {
   this.timeout(60_000);
 
   beforeEach(() => {
+    // https://github.com/GoogleChrome/lighthouse/issues/14572
+    expectError(/Request CacheStorage\.requestCacheNames failed/);
```
Comment (adamraine) [inline third-party/devtools-tests/e2e/lighthouse/navigation_test.ts:42]: drive-by for https://github.com/GoogleChrome/lighthouse/issues/14572

**Review by connorjclark — APPROVED** [review]
